src/components/earn/PoolCardTri/index.tsx

```diff
@@ -202,7 +219,12 @@ const StableStakingPoolCardTRI = (props: StablePoolCardTriProps) => {
           stakingInfo={stakingInfo}
         />
       )}
-      <DefaultPoolCardtri {...props} enableClaimButton={amountIsClaimable} enableModal={enableModal} />
+      <DefaultPoolCardtri
+        {...props}
+        enableClaimButton={amountIsClaimable}
+        enableModal={enableModal}
+        stakingInfo={stakingInfo}
```

Comment:
See above

```suggestion
```

Reply:
Answered in: 
 https://github.com/trisolaris-labs/interface/pull/374#discussion_r956681418